website/docs/getting-started/for-designers.md

```diff
@@ -208,4 +208,4 @@ Explore more content about libraries, components, and styles directly from Figma
 
 ## Support
 
-If you have questions or need assistance using Helios libraries, <LinkTo class="doc-link-generic" @route="show" @model="about/support">reach out for support</LinkTo>.
+<LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Helios Design Systems Team</LinkTo> for support if you have questions or need assistance using Helios libraries.
```

Comment:
I don't think the order of the sentence needed to change here, just the language used. 
```suggestion
If you have questions or need assistance using Helios libraries, <LinkTo class="doc-link-generic" @route="show" @model="about/support">contact the Design Systems Team</LinkTo>.
```